src/renderer/modules/webpack/index.ts

```diff
@@ -380,6 +380,29 @@ export namespace filters {
       return typeof match === "string" ? source.includes(match) : match.test(source);
     };
   };
+
+  /**
+   * Get a module that has the given value on one of its exports
+   * @param match The string to check the value against
+   *
+   * @example
+   * ```
+   * const classes = getModule(filters.byValue("container-2sjPya"));
+   * ```
+   *
+   * @remarks
+   * Great for getting a module for a specific classname
+   */
+  export const byValue = (match: string | RegExp) => {
+    return (m: RawModule) => {
+      if (!m.exports) {
+        return false;
+      }
+      return typeof match === "string"
```

Comment:
module exports is not guaranteed to be an object, so you should check for that here